scripts/dotnet.py

```diff
@@ -568,6 +601,14 @@ def __process_arguments(args: list):
         help='Download DotNet Cli from the Channel specified.'
     )
 
+    install_parser.add_argument(
+        '--version',
```

Comment:
should this also become `--dotnet-version`?

Reply:
meant to remove that